src/python/bot/minimizer/html_minimizer.py

```diff
@@ -190,12 +190,12 @@ def get_tokens_and_metadata(data):
   @staticmethod
   def combine_worker_tokens(tokens, prefix='', suffix=''):
     """Combine tokens for a worker minimizer."""
-    return '%s%s%s' % (prefix, ''.join(tokens), suffix)
+    return prefix + b''.join(tokens) + suffix
```

Comment:
prefix, suffix need to have b'' ?

Reply:
The values passed are output from combine_tokens, so we should be safe. I'd prefer to avoid explicitly converting to bytes here.

Reply:
i mean the default values of prefix and suffix are not bytes, can you change those.

Reply:
Ah, missed that. Done!